packages/turf-collect/LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Morgan Herlocker
+Copyright (c) 2016 Rowan Winsemius
```

Comment:
Why remove Morgan? He was the creator of `turf-collect`, you shouldn't be replacing your name for his.